wewatch-client/src/components/Match/Match.js

```diff
@@ -27,48 +27,48 @@ const Match = (props) => {
-        setNewRoomId(newRoomId)
-        setTryAgain(true);
-    });
-    return () => { socket.off('tryAgainRedirectUser')};
+    useEffect(() => {
+        socket.on('tryAgainRedirectUser', ({newRoomId}) => {
+            setNewRoomId(newRoomId)
+            setTryAgain(true);
+        });
+        return () => { socket.off('tryAgainRedirectUser')};
     })
 
     return (
         <Layout>
             <Container>
                 <Col>
                     <Row>
-                        <div class='center'>
-                        { tryAgainHost ? <Redirect to={`/lobby?name=${name}&reset=${true}&oldRoomId=${roomId}`}/> : null}
-                        { tryAgain ? <Redirect to={`/lobby?name=${name}&room=${newRoomId}`}/> : null}
-                        <h1>It's a match!</h1>
-                        <span>Everyone wants to watch:</span>
+                        <div class="text-container center">
+                            { tryAgainHost ? <Redirect to={`/lobby?name=${name}&reset=${true}&oldRoomId=${roomId}`}/> : null}
+                            { tryAgain ? <Redirect to={`/lobby?name=${name}&room=${newRoomId}`}/> : null}
+                            <h1>It's a match!</h1>
+                            <span>Everyone wants to watch:</span>
                         </div>
-                        </Row>
-                        <Row>
+                    </Row>
+                    <Row>
                         <StyledDiv alignItems="center" flexDirection="column">
                             <StyledDiv padding={2}>
-                                <MovieDetail
-                                title={movieData.title}
-                                year={movieData.released}
-                                lengthOfMovie={movieData.runtime}
-                                rating={movieData.rating}
-                                mediaType={movieData.type}
-                                imageURL={movieData.image}
-                                description={movieData.synopsis}
+                                <MatchMovieDetail
+                                    title={movieData.title}
+                                    year={movieData.released}
+                                    lengthOfMovie={movieData.runtime}
+                                    rating={movieData.rating}
+                                    mediaType={movieData.type}
+                                    imageURL={movieData.image}
+                                    description={movieData.synopsis}
                                 />
                             </StyledDiv>
-                            { isHost ? 
-                                <StyledDiv height="50%" marginHorizontal={7}>
-                                <button className={'button mt-20'} type="button" onClick={tryAgainEmit}>Try Again</button>
+                            {isHost ?
+                                <StyledDiv height="100px" marginHorizontal={7}>
```

Comment:
if we take out the percentages and hardcode the px here, the height won't be sizeable to different screens. That being said, do we want the movie details component to be the same size throughout for all screens for this application?

Reply:
I think for the cases of this demo, we are presenting it as a web application. I think that we can take a look at responsiveness at a later time but I think this change makes sense for this sprint. :)